driver/pelican/interface.md

```diff
@@ -0,0 +1,98 @@
+  Dtstart: time.Date(0, 0, 0, hour, minute, 0, 0, timezone),
+})
+```
+
+Three fields are configured.
+- Frequency indicates the interval with which this event occurs.
+- Wkst tells us which day of the week (Sunday - Saturday) this event occurs.
+- Dtstart is a required field that indicates the "start date" of the particular event. In Go, the Dtstart field is a time.Date object, which is initialized with the following parameters: year, month, day, hour minute, second, millisecond, timezone. For our purposes, there is no real concept of a "start date", just the time, so the year, month, and day parameters are filled with dummy values of 0. Only hour, minute, and timezone (which can be determined from the Pelican settings + schedule) are filled in. As long as an individual knows the time is in RRule format, he or she will be able to determine each field.
+
+The translation from the above RRule format to a string is performed using the RRule-go module, specifically this function linked here:
+https://github.com/teambition/rrule-go/blob/master/str.go#L123
+
+Within the thermSchedule.go code, the last line of the convertTimeToRRule function calls the ".string()" function of the RRule object. The implementation of this function is fairly straightforward. In a nutshell, the conversion function scans through each of the RRule object's fields. The "key-value" pair of each field is appended to a string object, which is ultimately returned. Some helper functions are used primarily for casting a variety of types into a string, such as appendIntsOption, timeToStr (for Dtstart), and append. In a general sense, the conversion function is meant to achieve two things:
+
+1. Be as human readable as possible. If one reads the resulting string output, he or she should easily be able to identify the interval, frequency, count, and start/end dates of the respective event.
+2. Can be converted back into RRule format. The RRule-go module has a complementary ".StrToRRule(rfcString string)" function that converts from string back to an RRule object.
+
+##### Time Format Conversion Examples
+
+The following is an example of what we can expect the conversion to take in and output for different types of events with different settings
+
+With the Pelican Thermostats, we will typically have the following settings:
+
+```
+Frequency: Weekly
+Wkst: Day of Week (Sunday, Monday...Saturday)
+Dtstart: time.Date(0, 0, 0,  Hour, Minute, 0, 0, Timezone)
+```
+
+As a reminder, the parameters of the time.Date object are Year, Month, Date, Hour, Minute, Second, Millisecond, and Timezone. Frequency is always set to "Weekly". Wkst and the Hour/Minute/Timezone depend on the value that is being retrieved in addition to the Pelican's timezone. The NewRRule object's fields are populated with these values and the aforementioned ".String()" method is called. Assuming the Wkst is Sunday and Dtstart is 6:00 a.m. with a U.S. Pacific Timezone, the output will look like this:
```

Comment:
Clarify that the timezone here is Pacific *Standard* Time

Reply:
Got it, rephrased it to clarify that!